R/infrastructure.R

```diff
@@ -33,6 +39,12 @@ use_testthat <- function(pkg = ".") {
     pkg = pkg
   )
 
+  if (use_catch) {
```

Comment:
This won't get run if `testthat` infrastructure (minus Catch) has already been initialized (e.g. you can't run `devtools::use_testthat(catch = TRUE)` to add Catch to a package already using `testthat`).

Could that be alleviated? E.g. move this to a separate function, `use_catch()`), and call it earlier in the `use_testthat()` call?